app/lib/fields/builtin_field.rb

```diff
@@ -8,14 +8,14 @@ def name=(val)
     @name = convert_name(val)
   end
 
-  def input(builder)
+  def input(builder, options = builder_options)
     # this is EXTREMELY dangerous
     # formtastic is getting reflection of association to get countries list
     # BUT it needs symbol, and when we give it a symbol, they can get any association of object
     #
     # so thats why it limits converting to symbol to just some attributes
     # and of course symbols are not GC'ed so DOS possiblity
```

Comment:
hmm, I think this comment is not correct anymore

https://bugs.ruby-lang.org/issues/9634

Reply:
OK!